.claude/skills/test-assess/SKILL.md

```
@@ -0,0 +1,94 @@
+
+Shallow clone each repo into that directory:
+
+```
+git clone --depth 1 <repo-url> $TESTDIR/<repo-name>
+```
+
+### 5. Run assessments
+
+Run each assessment using the local checkout, not the globally installed package:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)